Styles/styles-beach.css

```
@@ -0,0 +1,128 @@
+@import url(styles-global.css);
+
+.header-item {
+    background-image: url(../Beach-Assets/header-image.jpg);
+}
+
+.header-text {
+    min-width: 70%;
+}
+
+.paragraph-style {
+    margin: auto;
+}
+
+:is(.image1, .image2) {
+    display: flex;
+    width: 92vw;
+    border-radius: 1.25rem;
+    object-fit: cover;
+    margin-inline-start: auto;
+    margin-inline-end: auto;
```

Poți simplifica aici, fiind aceeași valoare.

```suggestion
    margin-inline: auto;
```

---

Chiar dacă am atras atenția aici, problema apare și în alte părți.